src/main/java/auth/TdpRecruitmentUnauthorizedHandler.java

```diff
@@ -0,0 +1,25 @@
+package auth;
+
+import io.dropwizard.auth.UnauthorizedHandler;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.Response;
+
+/**
+ * Created by dawidkw on 22/08/16.
+ */
```

Comment:
We know, it all will be changed, including removing comments and changing how the view looks.